# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ScriptKind.h
# Commit: b8ecf23519c9
# Full Hash: b8ecf23519c9ab5180d08f00c712674bfe968dac
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r 85fd9fc9cffa -r b8ecf23519c9 js/loader/ScriptKind.h
--- a/js/loader/ScriptKind.h	Mon Feb 21 15:39:08 2022 +0000
+++ b/js/loader/ScriptKind.h	Mon Feb 21 15:39:09 2022 +0000
@@ -4,15 +4,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef mozilla_dom_ScriptKind_h
-#define mozilla_dom_ScriptKind_h
+#ifndef js_loader_ScriptKind_h
+#define js_loader_ScriptKind_h
 
-namespace mozilla {
-namespace dom {
+namespace JS::loader {
 
 enum class ScriptKind { eClassic, eModule, eEvent };
 
-}  // namespace dom
-}  // namespace mozilla
+}  // namespace JS::loader
 
 #endif